<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pro Muxer - Professional Video Stream Muxing Tool</string>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <!-- File Operations Group -->
     <widget class="QGroupBox" name="fileGroup">
      <property name="title">
       <string>File Operations</string>
      </property>
      <layout class="QHBoxLayout" name="fileButtonLayout">
       <item>
        <widget class="QPushButton" name="addFilesBtn">
         <property name="text">
          <string>Add Files</string>
         </property>
         <property name="toolTip">
          <string>Add individual video files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addFolderBtn">
         <property name="text">
          <string>Add Folder</string>
         </property>
         <property name="toolTip">
          <string>Add all video files from a folder</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeBtn">
         <property name="text">
          <string>Remove Selected</string>
         </property>
         <property name="toolTip">
          <string>Remove selected files from the list</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearBtn">
         <property name="text">
          <string>Clear All</string>
         </property>
         <property name="toolTip">
          <string>Clear all files from the list</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="fileButtonSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <!-- File Table -->
     <widget class="QTableWidget" name="fileTable">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>File Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Video Codec</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Resolution</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Frame Rate</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Bit Depth</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Color Space</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Duration</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>File Size</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Output Name</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <!-- Settings Group -->
     <widget class="QGroupBox" name="settingsGroup">
      <property name="title">
       <string>Output Settings</string>
      </property>
      <property name="maximumHeight">
       <number>120</number>
      </property>
      <layout class="QGridLayout" name="settingsLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="outputFolderLabel">
         <property name="text">
          <string>Output Folder:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="outputFolderEdit">
         <property name="toolTip">
          <string>Directory where processed files will be saved</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="browseBtn">
         <property name="text">
          <string>Browse</string>
         </property>
         <property name="maximumWidth">
          <number>80</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="formatLabel">
         <property name="text">
          <string>Output Format:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="formatCombo">
         <property name="toolTip">
          <string>Target container format</string>
         </property>
         <item>
          <property name="text">
           <string>mp4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>mkv</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>mov</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>webm</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ts</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="namingLabel">
         <property name="text">
          <string>Naming Rule:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="namingLayout">
         <item>
          <widget class="QLineEdit" name="prefixEdit">
           <property name="placeholderText">
            <string>Prefix</string>
           </property>
           <property name="toolTip">
            <string>Text to add before original filename</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="namingMiddleLabel">
           <property name="text">
            <string>[filename]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="suffixEdit">
           <property name="placeholderText">
            <string>Suffix</string>
           </property>
           <property name="toolTip">
            <string>Text to add after original filename</string>
           </property>
           <property name="text">
            <string>_muxed</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="conflictCombo">
         <property name="toolTip">
          <string>How to handle existing output files</string>
         </property>
         <item>
          <property name="text">
           <string>Auto Rename</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Overwrite</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Skip</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QCheckBox" name="compatibilityCheck">
         <property name="text">
          <string>Smart Compatibility</string>
         </property>
         <property name="toolTip">
          <string>Automatically choose compatible container formats</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <!-- Processing Controls Group -->
     <widget class="QGroupBox" name="processGroup">
      <property name="title">
       <string>Processing</string>
      </property>
      <property name="maximumHeight">
       <number>80</number>
      </property>
      <layout class="QHBoxLayout" name="processLayout">
       <item>
        <widget class="QPushButton" name="startBtn">
         <property name="text">
          <string>Start Processing</string>
         </property>
         <property name="toolTip">
          <string>Begin batch muxing process</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopBtn">
         <property name="text">
          <string>Stop</string>
         </property>
         <property name="toolTip">
          <string>Stop current processing</string>
         </property>
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="toolTip">
          <string>Overall processing progress</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="statusLabel">
         <property name="text">
          <string>Ready</string>
         </property>
         <property name="minimumWidth">
          <number>200</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <!-- Log Group -->
     <widget class="QGroupBox" name="logGroup">
      <property name="title">
       <string>Log Output</string>
      </property>
      <property name="maximumHeight">
       <number>200</number>
      </property>
      <layout class="QVBoxLayout" name="logLayout">
       <item>
        <layout class="QHBoxLayout" name="logControlLayout">
         <item>
          <widget class="QCheckBox" name="infoCheck">
           <property name="text">
            <string>Info</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="warningCheck">
           <property name="text">
            <string>Warning</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="errorCheck">
           <property name="text">
            <string>Error</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="logControlSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="clearLogBtn">
           <property name="text">
            <string>Clear Log</string>
           </property>
           <property name="maximumWidth">
            <number>80</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="logEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>Processing log with different message levels</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <!-- Simplified UI: Removed File and Help menus as requested in bug_report.md -->
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>